includes/functions/functions_email.php

```diff
@@ -191,6 +191,7 @@ function zen_mail($to_name, $to_address, $email_subject, $email_text, $from_emai
       $zco_notifier->notify('NOTIFY_EMAIL_AFTER_EMAIL_FORMAT_DETERMINED');
 
       // now lets build the mail object with the phpmailer class
+      require DIR_APP_LIBRARY . 'PHPMailer/src/PHPMailer.php'; 
```

Comment:
@zcwilt shouldn't the class autoloader take care of this, so we don't need to do a manual `require` here?

Reply:
includes/auto_loaders/config.core.php 241-248 should be updated in any event.

Reply:
But I saw other manual requires for classes such as in includes/application_top.php: 
`DIR_APP_LIBRARY . 'aura/autoload/src/Loader.php';`

Reply:
I know for a fact that what's in this PR does work, because I had to test it a bunch of times to get it to work.

Reply:
Will merge for now. May update later to make less rigid. (Preferring to allow class autoloaders do their own requires)